pkgs/applications/misc/electrum/default.nix

```diff
@@ -1,23 +1,30 @@
-{ stdenv, fetchurl, buildPythonPackage, slowaes, ecdsa, pyqt4 }:
+{ stdenv, fetchurl, buildPythonPackage, pythonPackages, slowaes }:
 
 buildPythonPackage rec {
   namePrefix = "";
   name = "electrum-${version}";
-  version = "1.9.8";
+  version = "2.0.3";
 
   src = fetchurl {
     url = "https://download.electrum.org/Electrum-${version}.tar.gz";
-    sha256 = "8fc144a32013e4a747fea27fff981762a6b9e14cde9ffb405c4c721975d846ff";
+    sha256 = "1kzrbnkl5jps0kf0420vzpiqjk3v1jxvlrxwhc0f58xbqyc7l4mj";
   };
 
-  buildInputs = [ slowaes ecdsa ];
-
-  propagatedBuildInputs = [
-    slowaes
+  buildInputs = with pythonPackages; [
+    dns
     ecdsa
-    pyqt4
+    pbkdf2
+    protobuf
+    pyasn1
+    pyasn1-modules
+    qrcode
+    requests
+    slowaes
+    tlslite
   ];
 
+  propagatedBuildInputs = with pythonPackages; buildInputs ++ [ pyqt4 ];
```

Comment:
Without it I get "no module named foo" errors at runtime. I initially tried without propagation, but after having had to add 5-6 of the inputs to propagatedInputs, I decided to simply add all of them.
Note that input propagation is present in the old version as well, it's not something I've added, though I tried to get rid off it.